ProfileView/src/[sandbox]/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Remember we talked about having a single sandbox in the root? It can be a follow up PR, but i think this will be useful

Reply:
Yep, I want to discuss our approach here but I agree